credits.md

```diff
@@ -17,3 +17,4 @@
 - [CI greetings Bot](https://github.com/Ansub/SyntaxUI/pull/182) - [Daksh Singh Rathore](https://twitter.com/dakshsinghrath9)
 - [Issue Template](https://github.com/Ansub/SyntaxUI/pull/72) - [Daksh Singh Rathore](https://twitter.com/dakshsinghrath9)
 - [Automated Issue labeling](https://github.com/Ansub/SyntaxUI/pull/78) - [Daksh Singh Rathore](https://twitter.com/dakshsinghrath9)
+- [Icon Toggle](https://syntaxui.com/components/toggle/icon-toggle) - [Shashank Verma](https://x.com/Shashank_V99)
```

Comment:
I apologize, but there are no credits for code rewrites, as the person responsible is a maintainer, not the creator.

Reply:
Thanks for the feedback. Updated the Credits.md file and removed the maintainer credits.